src/components/common/Loading.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+import Spinner from './icons/Spinner';
+import { breakpoint } from '../../styles/globalStyles';
```

Comment:
You don't appear to be using `breakpoint` in this file

Reply:
Done, thanks

Reply:
When inserted on a page, in mobile view, text not wrapping.